etc/docker/Dockerfile

```
@@ -39,11 +39,13 @@ ARG REFERENCE_DEVICE
 ARG RELEASE
 ARG REST_API
 ARG WEB_GUI
+ARG OTBR_MDNS
```

We don't use `OTBR_` prefix for other Dockerfile ARGs. Should we be consistent?

```suggestion
ARG MDNS
```